README.md

```diff
@@ -172,4 +173,14 @@ After you create a development app (`bundle exec rake development_app`):
 
 - [Social providers integration](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/docs/social_providers.md): Enable sign up from social networks.
 
+## Technical tradeoffs
+
+### Turbolinks
+
+Decidim doesn't support `turbolinks` so it isn't include on our generated apps and it's removed for existing Rails applications which install the Decidim engine.
+
+The main reason for this is we are injecting some scripts into the body for some individual pages and Turbolinks loads the scripts in parallel. For some libraries like ![leaflet](http://leafletjs.com/) it's very inconvenient because its plugins extend an existing global object.
+
+The support of Turbolinks was dropped in #616.
```

Comment:
Add: "If you're interested in bringing turbolinks back, further discussion is welcome."